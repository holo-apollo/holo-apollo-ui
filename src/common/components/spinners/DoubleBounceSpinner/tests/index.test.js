import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import DoubleBounceSpinner from '../index';

describe('DoubleBounceSpinner', function() {
  it('should render normally', function() {
    const wrapper = mount(<DoubleBounceSpinner />);
    expect(wrapper.exists()).toBe(true);
  });

  it('should match snapshot', function() {
    const tree = renderer.create(<DoubleBounceSpinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
