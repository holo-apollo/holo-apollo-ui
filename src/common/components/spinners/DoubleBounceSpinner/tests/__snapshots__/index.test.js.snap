// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoubleBounceSpinner should match snapshot 1`] = `
<div
  className="DoubleBounceSpinner__Spinner-kwvmmk-0 kSrIIq"
  size={40}
>
  <div
    className="DoubleBounceSpinner__DoubleBounce-kwvmmk-1 gjvNJt"
    color="#4caad0"
  />
  <div
    className="DoubleBounceSpinner__DoubleBounce-kwvmmk-1 hvhqxi"
    color="#4caad0"
  />
</div>
`;
